bundles/org.openhab.binding.melcloud/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -123,10 +123,17 @@
 	<channel-type id="tankWaterTemperature-channel">
 		<item-type>Number:Temperature</item-type>
 		<label>Tank Temperature</label>
-		<description>Temperature of water i</description>
+		<description>Temperature of water</description>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
+	<channel-type id="tankTargetWaterTemperature-channel">
+		<item-type>Number:Temperature</item-type>
+		<label>Tank target temperature</label>
+		<description>Setpoint temperature of water</description>
+		<state min="20" max="65" step="1" pattern="%.0f %unit%" readOnly="false"/>
```

Comment:
Can you add semeantic tags for all added channel types?

Point + Property

Applies to all added channels